src/services/logIn.jsx

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
```

Comment:
just a suggestion- it's good to name components in .jsx, all the helper functions should be named in .js extension. It makes the file structure more readable.